### PR TITLE
Satellitedata

### PR DESCRIPTION
**Pull request by SoyGema**
CONTEXT : 
Add satellite data tracked with dvc for [workshop](https://github.com/iterative/VSCode-DVC-Workshop)! 

Configuration for  all for the workshop assistants to do `dvc get-url https://github.com/iterative/dataset-registry/workshopdvcdata/ `and have the data

Thanks in advance

### CODE REVIEW

File: .dvc/config [hunk 1]
```diff
@@ -2,3 +2,5 @@
     remote = storage
 ['remote "storage"']
     url = https://remote.dvc.org/dataset-registry
+['remote "local"']
```
Comment (shcheklein) [inline .dvc/config:5]: please remove this

File: satellitedata/satellitedata.dvc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+outs:
```
Comment (shcheklein) [inline satellitedata/satellitedata.dvc:1]: Q: why do we have two .dvc files?

File: satellitedata/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.env/
+/satellitedata
+/jan_train.csv
```
Comment (shcheklein) [inline satellitedata/.gitignore:3]: Q: what is this file?

**Review by shcheklein — CHANGES_REQUESTED** [review]
It seems some cleanup is needed.

Q: Did you manage to `push` data to the `dataset-registry` location on S3?

File: .dvc/config (hunk 1 above)
Comment (SoyGema) [reply .dvc/config:5]: Removed.
Included also line 6 url
Remote test in local

File: satellitedata/.gitignore (hunk 3 above)
Comment (SoyGema) [reply satellitedata/.gitignore:3]: A test. removed

**Comment by SoyGema** [comment]
sending it again due some issues with files

**Comment by shcheklein** [comment]
@SoyGema it is closed, is it intentional?

**Comment by SoyGema** [comment]
yes sorry I will do this tomorrow. Feeling tired and was doing other workshop stuff !

**Pull request closed without merging**